OpenRA.Game/Network/Session.cs

```diff
@@ -138,6 +138,7 @@ public static Client Deserialize(MiniYaml data)
 			public string Location;
 
 			public ClientState State = ClientState.Invalid;
+			public int Handicap = 100; // could be 50, 60, 70, 80, 90 or 100
```

Comment:
The upside of using a player trait is that you can expose these to a trait info field, so they aren't hard-coded values.